swarm/pss/pss.go

```diff
@@ -472,7 +465,19 @@ func (p *Pss) process(pssmsg *PssMsg, raw bool, prox bool) error {
 	p.executeHandlers(psstopic, payload, from, raw, prox, asymmetric, keyid)
 
 	return nil
+}
+
+// copy all registered handlers for respective topic in order to avoid deadlock
+func (p *Pss) getHandlers(topic Topic) map[*handler]bool {
```

Comment:
Please, return with `*handler[]` as we don't use the `bool` value in the single place of use.